aixplain/modules/model.py

```diff
@@ -54,6 +56,7 @@ def __init__(
         description: Text = "",
         api_key: Optional[Text] = None,
         url: Text = config.MODELS_RUN_URL,
+        backend_url: Text = config.BACKEND_URL,
```

Comment:
why the backend needs to be received as an input parameter. It never changes, right?

Reply:
Yes, I removed the `url` and `backend_url` parameters from this constructor.